app/models/api_token.rb

```diff
@@ -8,6 +8,24 @@ class APIToken < ApplicationRecord
   has_many :users, through: :organizations
   has_many :publications, through: :users
```

Comment:
```ruby
has_many :users, through: :organizations
has_many :publications, through: :users
```

Do we still need these two associations? Are they used elsewhere?

Reply:
Doesn't look like it.  I'll remove them and see what happens.